### PR TITLE
update from 2.2.5 to 2.2.6

### PR DESCRIPTION
**Pull request by xingma**
* [X] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/circexplorer2/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
   name: circexplorer2
-  version: "2.2.5"
+  version: "2.2.6"
 
 source:
-  fn: CIRCexplorer2-2.2.5.tar.gz
-  url: https://pypi.python.org/packages/f2/df/7c4aa81d76a097fb6cd9b0031900899da353dffd7195e555ef1db4065bdf/CIRCexplorer2-2.2.5.tar.gz
-  md5: 3269f36ef08b6e2ed83e9890567790cc
+  fn: CIRCexplorer2-2.2.6.tar.gz
+  url: https://pypi.python.org/packages/13/e4/d60ad898a2a5f7626bfae0052c77877cca734f6659ac3b417057c9f0fb60/CIRCexplorer2-2.2.6.tar.gz
```
Comment (bgruening) [inline recipes/circexplorer2/meta.yaml]: Just to double check, are all these build requirements really needed?

Comment (kepbod) [inline recipes/circexplorer2/meta.yaml]: Yes, CIRCexplorer depends on many softwares to work well.

Comment (bgruening) [inline recipes/circexplorer2/meta.yaml]: i know this is true at run-time but also at build-time?

Comment (kepbod) [inline recipes/circexplorer2/meta.yaml]: En, you are right. You mean only including essential staffs?

Comment (bgruening) [inline recipes/circexplorer2/meta.yaml]: Jupp :)

Comment (kepbod) [inline recipes/circexplorer2/meta.yaml]: OK, I only retain the necessary ones. Thank! :)

**Comment by kepbod** [comment]
ping @xingma

**Comment by kepbod** [comment]
I will recreate the RP. Close this one!

**Pull request closed without merging**